client-src/elements/chromedash-feature-page.js

```diff
@@ -447,9 +459,12 @@ export class ChromedashFeaturePage extends LitElement {
         .user=${this.user}
         ?canEdit=${this.userCanEdit()}
         .feature=${this.feature}
+        .gates=${this.gates}
         .comments=${this.comments}
         .process=${this.process}
-        .dismissedCues=${this.dismissedCues}>
+        .dismissedCues=${this.dismissedCues}
+        @open-approvals-event=${this.handleOpenApprovals}
```

Comment:
@open-approvals-event=${this.handleOpenApprovals} where is it being used?

Reply:
When the user clicks on a chromedash-gate-chip element, it generates an "open-approvals" event.  Those events are caught here and handleOpenApprovals() is called, which opens the approvals dialog.  Basically, dialog state is managed at the page level rather than having lower-level elements open a dialog directly.

Reply:
So this is an EventListener?

Reply:
Yep, the @ syntax is pretty handy in lit.
https://lit.dev/docs/components/events/#adding-event-listeners-in-the-element-template

Reply:
Aha thanks!